### PR TITLE
Complete the user guide for v1.2

### PR DESCRIPTION
**Pull request by sivayogasubramanian**
Changes:
- Added guides for booking related commands
- Fixed some errors made in https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/15

Note to reviewer: 
- The user guide website can be viewed [here](https://sivayogasubramanian.github.io/tp/UserGuide.html). This can be used rather than the git diff to check.

Closes https://github.com/AY2122S1-CS2103T-W16-2/tp/issues/13.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -115,11 +115,118 @@ Examples:
...
+
+Mark booking as done once the client has entered the gym on their time slot
+
+Format: `done booking d/DATE s/SLOT_NUMBER p/PHONE_NUMBER`
+
+<div markdown="block" class="alert alert-primary">
+
+**:bookmark: Note:**<br>
+
+* Date must follow the format specified [above](#features).
```
Comment (zognin) [inline docs/UserGuide.md:220]: Could you also add this point to the note for `add client` and `find client` just for consistency? Thank you

File: docs/UserGuide.md [hunk 2]
```diff
@@ -115,11 +115,118 @@ Examples:
...
+  e.g. <code>find booking n/Al</code> can show bookings for clients with names like Alice and Alex.
+
+* A booking either has no status (which represents a booking that is not completed) or a completed status.
+
+</div>
+
+Examples:
+* `find booking n/al` Find bookings for clients with names matching 'al'. E.g. Alex, Alice, Al.
+* `find booking d/12-12-2021 t/1930` Find all bookings on 12 December 2021 which starts at 1930 hrs.
+* `find booking s/c` Find all completed bookings.
```
Comment (zognin) [inline docs/UserGuide.md:202]: Could we indicate the possible values for STATUS? If `c` is completed, is `n` for not completed? Additionally, if we only have 2 statuses - completed or not completed, is it better to use a boolean attribute so we can have something like `c/YES_OR_NO`?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -144,16 +251,21 @@ If your changes to the data file makes its format invalid, BTBB will discard all
...
-**Delete client** | `delete client INDEX`
-**Find client** | `find client [n/KEYWORD] [p/KEYWORD] [e/KEYWORD] [a/KEYWORD] [b/KEYWORD] [v/YES_OR_NO] [m/PERIOD_TO_EXP]`<br> e.g. `find client n/al`
-**List client** | `list client`
-**Help** | `help`
+Action                   | Format, Examples
+-------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+**Add client**           | `add client n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/BIRTH_DATE v/YES_OR_NO m/START_DATE PERIOD` <br> e.g. `add client n/Alex Yeoh p/89653101 e/alexyeoh@gmail.com a/Choa Chu Kang St 62 Blk 123 #12-34 b/04031990 v/yes m/04062021 1m`
+**Delete client**        | `delete client INDEX`
+**Find client**          | `find client [n/KEYWORD] [p/KEYWORD] [e/KEYWORD] [a/KEYWORD] [b/KEYWORD] [v/YES_OR_NO] [m/PERIOD_TO_EXP]`<br> e.g. `find client n/al`
```
Comment (zognin) [inline docs/UserGuide.md:264]: I noticed the `find client` command format was changed above, could the changes be transferred here too?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -60,7 +60,7 @@ If you are looking to keep your physique, down to your finger muscles, in shape,
 
 ### Adding a client: `add client`
```
Comment (zognin) [inline docs/UserGuide.md:61]: Hi I know the add client was not your part cause I forgot about it :D but under the note, is it possible to add a point like
`* PHONE_NUMBER is unique to a client. Each phone number in the system must belong to exactly one client.`

File: docs/UserGuide.md [hunk 5]
```diff
@@ -60,66 +60,163 @@ If you are looking to keep your physique, down to your finger muscles, in shape,
...
+### Deleting a booking: `delete booking`
+
+Deletes a booking from the application.
+
+Format: `delete booking INDEX`
+
+**Examples:**
+* `delete booking 1` Deletes the booking at index 1 in the booking list currently shown.
+
+### Finding booking by keywords: `find booking`
```
Comment (zognin) [inline docs/UserGuide.md:164]: Minor grammar change
```suggestion
### Finding bookings by keywords: `find booking`
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -60,66 +60,163 @@ If you are looking to keep your physique, down to your finger muscles, in shape,
...
+* The search is case-insensitive.
+
+* There must be 1 or more search arguments.
+
+* Partial search will be allowed. <br>
+  e.g. <code>find booking n/Al</code> can show bookings for clients with names like Alice and Alex.
+
+* <code>c/YES_OR_NO</code> represents the completion status of the booking.
+
+* <code>d/DATE</code> and <code>s/START_TIME</code> represents the booking date and time. It must follow the format specified [above](#features).
```
Comment (zognin) [inline docs/UserGuide.md:183]: Minor grammar change
```suggestion
* <code>d/DATE</code> and <code>s/START_TIME</code> represents the booking date and time. They must follow the format specified [above](#features).
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -60,66 +60,163 @@ If you are looking to keep your physique, down to your finger muscles, in shape,
...
+Format: `delete booking INDEX`
+
+**Examples:**
+* `delete booking 1` Deletes the booking at index 1 in the booking list currently shown.
+
+### Finding booking by keywords: `find booking`
+
+Finds booking whose attribute(s) matches the keyword(s).
+
+Format: `find booking [d/DATE] [s/START_TIME] [n/NAME] [c/YES_OR_NO]`
```
Comment (zognin) [inline docs/UserGuide.md:168]: Are we not adding phone number to the find params?
```suggestion
Format: `find booking [d/DATE] [s/START_TIME] [n/NAME] [p/PHONE_NUMBER] [c/YES_OR_NO]`
```

**Review by zognin — COMMENTED** [review]
Looks good other than a few minor things

**Review by Imranr2 — COMMENTED** [review]
LGTM! Just minor nits to consider

File: docs/UserGuide.md [hunk 8]
```diff
@@ -60,66 +60,166 @@ If you are looking to keep your physique, down to your finger muscles, in shape,
...
+
 * <code>v/YES_OR_NO</code> records the client's vaccination status.
 
-* <code>m/START_DATE PERIOD</code> records the client's membership status.<br>
-Start date is the start of the membership.<br>
-Period is the time from the start date that the membership is valid in months or years.<br>
-The period can be in number of months, suffixed by `m` or in years, suffixed by `y`.
+* <code>m/START_DATE</code> records the client's membership status. Start date is the start of the membership.
+
+* <code>pe/PERIOD</code> is the time from the start date that the membership is valid in months or years. The period can be in number of months, suffixed by `m` or in years, suffixed by `y`.
```
[the remark this answers is not in the document]
Comment (sivayogasubramanian) [reply docs/UserGuide.md:77]: @zognin What do you think?

[the remark this answers is not in the document]
Comment (zognin) [reply docs/UserGuide.md:77]: @sivayogasubramanian I think we can move since this period format occurs in more than 1 command.

**Review by Imranr2 — APPROVED** [review]
LGTM!

**Review by zognin — APPROVED** [review]
LGTM

**Pull request merged**